app/ai/voice/agents/breeze_buddy/template/hooks.py

```
@@ -243,6 +250,101 @@ async def execute(
+            executor = HttpRequestExecutor(session=context.aiohttp_session)
+
+            logger.info(
+                f"Executing HTTP {http_request.method.value} request to {http_request.url} "
+                f"for function '{function_name}'"
+            )
+
+            await executor.execute(config=http_request, resolved_fields=resolved_fields)
+
+            logger.info(f"HTTP request completed for function '{function_name}'")
+
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)